server/webapp/WEB-INF/rails.new/spec/controllers/cctray_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+##########################GO-LICENSE-START################################
+# Copyright 2015 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################GO-LICENSE-END##################################
+
+require File.join(File.dirname(__FILE__), "..", "spec_helper")
+
+describe CctrayController do
+  describe "routes" do
+    it "should resolve its route" do
+      expect(:get => '/new_cctray.xml').to route_to(:controller => "cctray", :action => "index", :format=> "xml")
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.